### PR TITLE
move linker to docker

### PR DESCRIPTION
**Pull request by generall**
fixes:

```
warning: /home/generall/projects/vector_search/qdrant/Cargo.toml: unused manifest key: profile.ci.linker
```

CC  @Jesse-Bakker

### CODE REVIEW

**Comment by Jesse-Bakker** [comment]
You'll probably want to add 
```yaml
- build-args |
     RUST_BUILD_PROFILE=ci
```

to the docker build steps in the workflow

**Comment by ffuugoo** [comment]
Very curious about low-ram test. It fails consistently on my #1594. 🥲

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -173,6 +173,8 @@ jobs:
             type=gha,scope=${{ github.base_ref }}
           cache-to: type=gha,mode=max,scope=${{ github.ref }}
           load: true
+          build-args: |
+            RUST_BUILD_PROFILE=ci
```
[the remark this answers is not in the document]
Comment (generall) [reply .github/workflows/integration-tests.yml:177]: it is there already

[the remark this answers is not in the document]
Comment (ffuugoo) [reply .github/workflows/integration-tests.yml:177]: Am I missing something?

Screenshot 2023-05-09 at 11 20 10

[the remark this answers is not in the document]
Comment (generall) [reply .github/workflows/integration-tests.yml:177]: huh

[the remark this answers is not in the document]
Comment (generall) [reply .github/workflows/integration-tests.yml:183]: I thought you were talking about `docker buildx build` inside scripts

**Review by ffuugoo — APPROVED** [review]
LGTM.

**Pull request merged**